src/test/java/mock/xmpp/MockXmppConnection.java

```diff
@@ -235,6 +235,16 @@ public IQ sendPacketAndGetReply(IQ packet)
         }
     }
 
+    @Override
```

Comment:
you can put empty docs tag here with {@inheritDoc} once they are filled in the interface

Reply:
i'll add this but no methods in this file have any javadocs it seems.

Reply:
turns out i didn't need this anymore